.github/workflows/test.yaml

```diff
@@ -0,0 +1,39 @@
+name: "Delta Flow Tests"
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        scala: [ 2.12.10 ]
```

Comment:
Spark runs on Scala 2.13 since [ver 3.2.1](https://spark.apache.org/docs/3.2.1/#downloading) also. It may be a good idea to add it to matrix.

Reply:
done